### PR TITLE
16.04 changes to /desktop/ubuntu-kylin

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Fixed up whitespace inconsistencies
- Made copy changes for 16.10 release

## QA

- Pull down changes
- Cross-ref copy changes with copy doc

Note: the layout of this page will look out of whack but @grahambancroft is fixing that upstream.

## Issue / Card

Copy doc: https://docs.google.com/document/d/1eJVoek0vT_vT4vy1Quw0NaBqm1fR9K2Dw8korIT2pKc/edit#

Trello:
https://trello.com/c/lTDdTeIk

### CODE REVIEW

File: templates/desktop/ubuntu-kylin.html [hunk 1]
```diff
@@ -6,72 +6,93 @@
...
-     			<li>New first time user guide</li>
-    		</ul>
-    		<ul class="list-ticks six-col last-col">
-     			<li>Support for the ARM64 chipset</li>
-     			<li>WeChat web application</li>
-    		</ul>
-    	</div>
-    	<div class="eight-col">
-     		<p><a href="https://wiki.ubuntu.com/{{latest_release_name}}/ReleaseNotes/UbuntuKylin" class="external">Read the full release notes</a></p>
+        <h2>What&rsquo;s new in Ubuntu {{lts_release_no_point}}?</h2>
```
Comment (anthonydillon) [inline templates/desktop/ubuntu-kylin.html]: This should be `What&rsquo;s new in Ubuntu {{lts_release_no_point}} LTS?`

File: templates/desktop/ubuntu-kylin.html [hunk 2]
```diff
@@ -149,39 +170,39 @@ <h2 class="six-col">A global partnership</h2>
...
-    		<h3>Unity 8 preview</h3>
-    		<p>Ubuntu Kylin {{latest_release_full}} offers an optional preview of Unity 8, the next generation Ubuntu user interface. To download the preview, search for unity8-desktop-session-mir in the Ubuntu Software Centre.</p>
-    		<p>Please note that it uses the current version of Mir, the new graphics platform being developed by the Ubuntu community. Mir only works with open-source graphics drivers at present.</p>
-    	</div>
+        <div class="five-col equal-height__item equal-height__align-vertically priority-0">
+            <img src="{{ ASSET_SERVER_URL }}1097135a-image-unity8.svg" alt="" width="299" height="213" />
+        </div>
+        <div class="seven-col equal-height__item last-col">
+            <h3>Unity 8 preview</h3>
+            <p>Ubuntu Kylin {{latest_release_full}} offers an optional preview of Unity 8, the next generation Ubuntu user interface. To download the preview, search for unity8-desktop-session-mir in the Ubuntu Software Centre.</p>
```
Comment (anthonydillon) [inline templates/desktop/ubuntu-kylin.html]: This should be the LTS variable. `Ubuntu Kylin 16.04 LTS offers an optional`

File: templates/desktop/ubuntu-kylin.html (hunk 1 above)
Comment (barrymcgee) [inline templates/desktop/ubuntu-kylin.html]: @anthonydillon There's a variable for that which I've now changed that to - `{{lts_release_full}}`

**Comment by barrymcgee** [comment]
@anthonydillon Ready for re-review..

**Pull request merged**